src/vm/threads.h

```diff
@@ -1509,6 +1509,7 @@ class Thread: public IUnknown
     Volatile<ULONG>      m_fPreemptiveGCDisabled;
 
     PTR_Frame            m_pFrame;  // The Current Frame
+    PTR_GCFrame          m_pGCFrame; // The topmost GC Frame
```

Comment:
I would put this more towards the end so that the asm offsets do not need to change.

Reply:
I have actually had it that way and moved it here during final cleanup feeling that it would be easier to find the member in WinDbg. But I don't feel strongly about it.